### DISCUSSION TITLE
Using Markdoc with Sourcehut/GitHub pages

### DISCUSSION THREAD

**Discussion by pariah-kite** [markdoc/markdoc#242] [Q&A] [answered]
Is there a way to set up a Markdoc powered site using [Sourcehut pages](https://srht.site/quickstart), or [GitHub pages](https://pages.github.com/)?

**Comment by mfix-stripe** [comment, marked as answer]
@pariah-kite yup! You can create a site just like ours (https://github.com/markdoc/docs), and then have it deploy to GitHub pages same way you would any other Next.js site. https://dev.to/github/how-to-host-a-static-nextjs-site-on-github-pages-4pe0
